etc/kayobe/inventory/group_vars/monitoring/firewall

```diff
@@ -28,29 +33,55 @@ firewallgen_ipv4_input_allow_rules:
     destination: "{{ internal_net_name | net_ip }}"
     comment: "hint: used by 'prometheus' in docker container 'prometheus_server'"
   - interface: "{{ internal_net_interface }}"
-    port: 9093
+    port: 9100
     proto: tcp
     destination: "{{ internal_net_name | net_ip }}"
-    comment: "hint: used by 'alertmanager' in docker container 'prometheus_alertmanager'"
+    comment: "hint: used by 'node_exporter' in docker container 'prometheus_node_exporter'"
   - interface: "{{ internal_net_interface }}"
-    port: 9100
+    port: 9108
     proto: tcp
     destination: "{{ internal_net_name | net_ip }}"
-    comment: "hint: used by 'node_exporter' in docker container 'prometheus_node_exporter'"
+    comment: "hint: used by 'elasticsearch_e' in docker container 'prometheus_elasticsearch_exporter'"
   - interface: "{{ internal_net_interface }}"
     port: 9198
     proto: tcp
     destination: "{{ internal_net_name | net_ip }}"
     comment: "hint: used by 'openstack-expor' in docker container 'prometheus_openstack_exporter'"
-  - interface: "all"
-    port: 880
-    proto: udp
-    comment: "hint: used by 'rpcbind'"
   - interface: "docker0"
     port: 123
     proto: udp
     destination: "172.17.0.1"
     comment: "hint: used by 'ntpd'"
+  - interface: "{{ internal_net_interface }}"
+    port: 9200
+    proto: tcp
+    destination: "{{ internal_net_name | net_ip }}"
+    comment: "hint: used by 'java' in docker container 'elasticsearch'"
+  - interface: "{{ internal_net_interface }}"
+    port: 9300
```

Comment:
I *think* this is for clustering so technically we don't need it because we have only one instance.